### DISCUSSION TITLE
Recover 6-8 months old unsaved snapchat messages??

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [rodit/SnapMod#301] [General]
I know it's not possible though until and unless snapchat gives me the data itself, but still is there any way out there I can at least see the messages which I didn't saved like around 6-8 months ago. I already saw MY DATA of Snapchat but they don't show unsaved chat messages.

**Comment by authorisation** [comment]
I'm not entirely sure about what you mean but messages are pulled from a server and if that server doesn't give you that information you simply can't
